projects/vdk-control-cli/src/vdk/internal/control/utils/output_printer.py

```diff
@@ -107,6 +108,40 @@ def print_dict(self, data: Dict[str, Any]) -> None:
             click.echo("{}")
 
 
+class MemoryPrinter(Printer):
+    output_buffer = io.StringIO()
```

Comment:
output_buffer is currently a class attribute (a static variable) which makes it shared across all instances of the class. It might be better to move it to an instance attribute so that each instance of MemoryPrinter has its own buffer. 
 
 In Python, this happens by defining a constructor using` __init__` 
 
 Read over https://docs.python.org/3/tutorial/classes.html

Reply:
fixed